Tests/test_geo.py

```diff
@@ -3165,6 +3164,91 @@ def test_soft_ex_affy_chp(self):
             self.assertEqual(len(record.col_defs), 0)
             self.assertEqual(len(record.table_rows), 0)
 
+    def test_record_str(self):
+        path = "Geo/GSM804.txt"
+        with open(path) as handle:
+            records = Geo.parse(handle)
+            record = next(records)
+            self.assertEqual(str(record), "GEO Type: SAMPLE\n"
+                                          "GEO Id: GSM804\n"
+                                          "Sample_author: Antoine,M,Snijders\n\n"
+                                          "Sample_author: Norma,,Nowak\n\n"
+                                          "Sample_author: Richard,,Segraves\n\n"
+                                          "Sample_author: Stephanie,,Blackwood\n\n"
+                                          "Sample_author: Nils,,Brown\n\n"
+                                          "Sample_author: Jeffery,,Conroy\n\n"
+                                          "Sample_author: Greg,,Hamilton\n\n"
+                                          "Sample_author: Anna,K,Hindle\n\n"
+                                          "Sample_author: Bing,,Huey\n\n"
+                                          "Sample_author: Karen,,Kimura\n\n"
+                                          "Sample_author: Sindy,,Law\n\n"
+                                          "Sample_author: Ken,,Myambo\n\n"
+                                          "Sample_author: Joel,,Palmer\n\n"
+                                          "Sample_author: Bauke,,Ylstra\n\n"
+                                          "Sample_author: Jingzhu,P,Yue\n\n"
+                                          "Sample_author: Joe,W,Gray\n\n"
+                                          "Sample_author: Ajay,N,Jain\n\n"
+                                          "Sample_author: Daniel,,Pinkel\n\n"
+                                          "Sample_author: Donna,G,Albertson\n\n"
+                                          "Sample_description: Coriell Cell Repositories "
+                                          "cell line <a h\n"
+                                          "ref=\"http://locus.umdnj.edu/nigms/nigms_cgi/display.cgi?GM05296\">GM05296</a>.\n\n"
+                                          "Sample_description: Fibroblast cell line derived "
+                                          "from a 1 mo\n"
+                                          "nth old female with multiple congenital malformations, "
+                                          "dysmorphic features, intr\n"
+                                          "auterine growth retardation, heart murmur, cleft palate, equinovarus deformity, \n"
+                                          "microcephaly, coloboma of right iris, clinodactyly, reduced RBC catalase activit\n"
```

Comment:
This line is overly long - indenting everything to the opening bracket of ``self.assertEqual(`` would be better (or adopt the ``black`` indentation style which would give even less indentation)